### ISSUE TITLE
PSR2 SwitchDeclaration rule possible false positive on break body indent

### ISSUE THREAD

**Issue by AshleyJSheridan** [squizlabs/PHP_CodeSniffer#713]
Give the code below (which seems valid as per the PSR2 spec https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#52-switch-case ) I'm getting errors from CS that the terminating statements for each `CASE` are not indented to the same level as the `CASE` body:

```php
switch($params)
{
	case (is_array($params) ):
		echo 'no';
		break;
	case 'wtf':
		echo 'wtf';
		break;
	default:
		echo 'do stuff';
		break;
}
```

I believe this is triggered by the first case (while it's unusual, it's valid and nothing in the PSR2 spec suggests it isn't) as it is using an expression of it's own, rather than a fixed string, or variable, for example.

**Comment by AshleyJSheridan** [comment]
Closed, it was related to another set of rules which were being triggered by my use of tabs instead of spaces to indent.
